src/docs.md

```diff
@@ -248,15 +248,28 @@ unsubscribe();
 
 <small>API: [CadenzaClient#createGeometry](./classes/CadenzaClient.html#createGeometry)</small>
 
-Edit a GeoJSON point geometry with a workbook map view in the background. The geometry coordinates are in the map's SRS (`useMapSrs: true`).
+Create a GeoJSON point geometry with a workbook map view in the background. The geometry coordinates are in the map's SRS (`useMapSrs: true`).
 
 ```javascript
 cadenzaClient.createGeometry('<embeddingTargetId>', 'Point', {
   useMapSrs: true,
 });
 
 cadenzaClient.on('editGeometry:ok', (event) => {
-  console.log('Geometry editing was completed', event.detail.geometry);
+  console.log('Geometry creation was completed', event.detail.geometry);
+});
+```
+
+_Note:_ Under the hood, creating a geometry is similar to editing a geometry.
+That's why the events use the `editGeometry` prefix.
+
+### Create a New Geometry with some reference layers on the background
+
+Create a GeoJSON polygon geometry with a workbook map view and some additional layers in the background.
+
+```javascript
+cadenzaClient.createGeometry('<embeddingTargetId>', 'Polygon', {
+  additionalLayers: [{"name":"layer_name_1","type":"geojson","content": {replace_with_geojson_FeatureCollection}},{"name":"layer_name_2","type":"geojson","content":{replace_with_geojson_FeatureCollection},{"name":"layer_name_3","type":"geojson","content":{replace_with_geojson_FeatureCollection}}]
```

Comment:
The other placeholders in here use <>. Proposal:
```markdown
#### Additional Background Layers

Create a GeoJSON polygon geometry with a workbook map view and some additional background layers.

```javascript
cadenzaClient.createGeometry('<embeddingTargetId>', 'Polygon', {
  additionalLayers: [
    { type: 'geojson', name: 'Example', content:  { <FeatureCollection> } },
    ...
  ]
});
```

(Fixes also the above "issues".)